Kitodo-Command/src/main/java/org/kitodo/command/Command.java

```diff
@@ -11,13 +11,102 @@
 
 package org.kitodo.command;
 
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.kitodo.api.command.CommandInterface;
 import org.kitodo.api.command.CommandResult;
 
 public class Command implements CommandInterface {
 
+    private static final Logger logger = LogManager.getLogger(Command.class);
+    private static final String CHARSET = "UTF-8";
+
+    /**
+     * Method executes a script.
+     *
+     * @param id
+     *            The id, to identify the command and it's results.
+     * @param command
+     *            The command as a String.
+     * @return The command result.
+     */
     @Override
     public CommandResult runCommand(Integer id, String command) {
-        throw new RuntimeException("Not implemented, yet");
+
+        CommandResult commandResult = null;
+        Process process = null;
+        String[] callSequence = command.split("[\\r\\n\\s]+");
+
+        try {
+            process = new ProcessBuilder(callSequence).start();
+            ArrayList<String> outputMessage = inputStreamArrayToList(process.getInputStream());
+            ArrayList<String> errorMessage = inputStreamArrayToList(process.getErrorStream());
+            int errCode = process.waitFor();
+
+            outputMessage.addAll(errorMessage);
+
+            commandResult = new CommandResult(id, command, errCode == 0, outputMessage);
```

Comment:
If a called script is returning with an error code unequal zero then are the error message(s) important and should be displayed in the frontend and maybe logged.

Reply:
I oversee that you added the error message in any case, so ignore my comment.